index.php

```diff
@@ -0,0 +1,64 @@
+<?php
+require_once __DIR__ . '/vendor/autoload.php';
+
+$db = new TopCat\Utilities\DB();
+$dbConnection = $db->dbConnect();
+
+// Prepare target strings
+$dropdownBreeds = '';
+$catshtml = '';
+
+// Populate the dropdown with the list of breeds from the DB
+$breedSql = $dbConnection->prepare('SELECT `breed` FROM `breed`');
+$breedSql->execute();
+$breeds = $breedSql->fetchAll();
```

Comment:
You shouldn't have your SQL in your html. Please make this either a method on an existing class or part of a new class. Wanna chat about the best place to put it?

Reply:
Yes, I'll chat about it